.travis.yml

```diff
@@ -8,6 +8,7 @@ dist: trusty
 jdk:
 - openjdk8
 install: true # skips travis' default installation step which executes gradle assemble.
+before_script: ./gradlew injectVersion
```

Comment:
Why `inject` and not just `setReleaseVersion`?

Reply:
168c067, ac9470f, 8379ce0